modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Select/BaseSelect.tsx

```diff
@@ -73,19 +91,57 @@ export function BaseSelect({
 			required={required}
 		>
 			<ClayAutocomplete>
-				<ClayAutocomplete.Input
-					defaultValue={value}
-					disabled={disabled}
-					onClick={() => setDropdownActive((active) => !active)}
-					placeholder={placeholder}
-					ref={inputRef}
-					value={value}
-					{...restProps}
-				/>
+				{useMultiSelect ? (
+					<ClayMultiSelect
```

Comment:
@danvb112 I think it would be better to pass all this logic (that is exclusive of MultiSelect) to the MultipleSelect component and keep the Base Select with only the common logic for both select types.

Reply:
Hey @naraandrade, since it is in BaseSelect that the component to make the input of the select is, I think this is the only place where we can make this condition, even if I move this logic inside the MultipleSelect component, I still have to import it in BaseSelect and use it pretty much the same as I did. The BaseSelect component already receives a child, related to dropdown items, so we have no way of passing other children to receive this ClayMuiltiSelect as input.

Reply:
Done!